### DISCUSSION TITLE
edge labels 显示结果异常

### DISCUSSION THREAD

**Discussion by qq296565890** [antvis/XFlow#76] [Q&A]
[【bug复现】](https://codesandbox.io/api/v1/sandboxes/define)
[](https://codesandbox.io/s/u6yov?file=/mock.ts)
打开mock.ts文件 并在120行后面加入如下代码：
`labels: [
      {
        position: { distance: 0.2 },
        attrs: { label: { text: "aaaaaaaaaa" } }
      },
      {
        position: { distance: 0.8 },
        attrs: { label: { text: "edge" } }
      }
    ]`
![7f4d9e861fa473c6533b7e9c09059e7](https://user-images.githubusercontent.com/26108264/147810269-25100c66-1481-40de-8cef-9c6ffce9cdad.png)

**Comment by qq296565890** [comment]
[bug复现地址]https://xflow.antv.vision/docs/tutorial/solutions/er
![image](https://user-images.githubusercontent.com/26108264/147810345-c7911ce7-669a-4b00-9362-11f739235867.png)
进入后打开mock.ts文件 并进行问题正文的操作即可复现

**Comment by lyn-boyu** [comment]
thanks for your report， we will solve this problem  ASAP
